app/store/index.js

```diff
@@ -187,6 +189,12 @@ export default function configureAppStore(initialState) {
             // check to see if the logout request was successful
             store.subscribe(async () => {
                 const state = store.getState();
+                const config = getConfig(state);
+
+                if (config?.SiteURL && getSiteUrl() !== config?.SiteURL) {
```

Comment:
Maybe this can just be
```suggestion
                if (getSiteUrl() !== config?.SiteURL) {
```

When we're repeating something using `?.`, that feels to me like we might be misusing it for some reason